modules/fishhook/1.0/fishhook.smk

```diff
@@ -116,23 +115,27 @@ rule _install_fishhook:
         input = CFG["logs"]["inputs"] + "install_fishhook.log"
     shell:
         """
-        R -q -e 'devtools::install_github(c("jokergoo/ComplexHeatmap","mskilab/gTrack", "mskilab/fishHook"))' >> {log.input} &&
-        touch {output.complete}"""
-
-
-# Example variant calling rule (multi-threaded; must be run on compute server/cluster)
-rule _run_fishhook:
+        R -q -e 'devtools::install_github("mskilab/gUtils")' >> {log.input} &&
```

Comment:
so the Complexheatmap is not required?

Reply:
I don't believe so. It may have been that the developers removed it's dependency. I followed their installation instructions [here](https://github.com/mskilab-org/fishHook?tab=readme-ov-file#installation) which don't mention it. 

Just to double check, I found and activated the fishhook conda env that snakemake created when running this through GAMBL.
- `conda activate <env named by md5sum>`
- `which R` shows the one in this env
- R
    - `find.package("ComplexHeatmap")` gives an error 
    ```
    Error in find.package("ComplexHeatmap") : 
        there is no package called ‘ComplexHeatmap’
    ```
    - `library(ComplexHeatmap)` gives an error
    ```
    Error in library(ComplexHeatmap) : 
        there is no package called ‘ComplexHeatmap’
    ```

**Conclusion**: Package most likely not needed